### PR TITLE
[WIP][10.0][ADD] Module l10n_ch_evat

### PR DESCRIPTION
**Pull request by grindtildeath**
**Work in progress**

This module allows to generate E-VAT Declaration in XML to be transmitted to the swiss Federal Tax Administration.

### CODE REVIEW

**Review by tschanzt — APPROVED** [review]
Only skimmed through but looks good.

File: l10n_ch_evat/data/ir_sequence.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="seq_l10n_ch_evat" model="ir.sequence">
+        <field name="name">E-VAT Report Business Reference Id</field>
+        <field name="code">l10n_ch.evat.report</field>
+        <field name="prefix">EVAT-</field>
+        <field name="padding">5</field>
+    </record>
+</odoo>
```
Comment (leemannd) [inline l10n_ch_evat/data/ir_sequence.xml:9]: missing newline EOF

File: l10n_ch_evat/models/evat_xml_report.py [hunk 2]
```diff
@@ -0,0 +1,665 @@
...
+
+    name = fields.Char(required=True)
+    date_from = fields.Date(default=lambda e: e._default_date_from())
+    date_to = fields.Date(default=lambda e: e._default_date_to())
+    generation_date = fields.Date(readonly=True)
+    target_moves = fields.Selection(TARGET_MOVES_SELECTION, required=True,
+                                    default='posted')
+    state = fields.Selection([('draft', 'Draft'), ('generated', 'Generated')],
+                             default='draft', required=True)
+    # TODO translate labels (Caution legal)
```
Comment (leemannd) [inline l10n_ch_evat/models/evat_xml_report.py:112]: Hum, do we want to address this?

**Review by leemannd — APPROVED** [review]
Got a few comments.
I think we can get rid of a part of the errors of the linter if we rebase it

**Comment by yvaucher** [comment]
Please rebase

**Pull request closed without merging**